### PR TITLE
(PC-28940) feat(home): add venue map block on home

### PR DESCRIPTION
**Pull request by voisinhugo**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-28940

## Checklist

I have:

- [x] Made sure my feature is working on the relevant real / virtual devices (native and web).
- [x] Written **unit tests** native (and web when implementation is different) for my feature.
- [x] Added a **screenshot** for UI tickets or deleted the screenshot section if no UI change
- [ ] If my PR is a bugfix, I add the link of the "résolution de problème sur le bug" [on Notion][1]

## Screenshots

| Platform         | Mockup/Before | After |
| :--------------- | :-----------: | :---: |
| iOS              |               |  image |
| Desktop - Chrome |               | image |

[1]: https://www.notion.so/passcultureapp/R-solution-de-probl-mes-sur-les-bugs-5dd6df8f6a754e6887066cf613467d0a

### CODE REVIEW

File: src/features/home/components/modules/VenueMapModule.native.test.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+}))
+
+describe('VenueMapModule', () => {
+  it('should render venue map block when user is located and feature flag enabled', () => {
+    render(<VenueMapModule />)
+
+    expect(screen.getByText('Carte des lieux culturels')).toBeOnTheScreen()
+  })
+
+  it('should render venue map block when feature flag is disabled', () => {
```
Comment (cedricls1) [inline src/features/home/components/modules/VenueMapModule.native.test.tsx:36]: `should not`

File: src/features/home/components/modules/VenueMapModule.native.test.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    useFeatureFlagSpy.mockReturnValueOnce(false)
+    render(<VenueMapModule />)
+
+    expect(screen.queryByText('Carte des lieux culturels')).not.toBeOnTheScreen()
+  })
+
+  it('should render venue map block when user is not located', () => {
```
Comment (cedricls1) [inline src/features/home/components/modules/VenueMapModule.native.test.tsx:43]: `should not`

File: src/features/home/components/modules/VenueMapModule.native.test.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      hasGeolocPosition: false,
+      selectedLocationMode: mockSelectedLocationMode,
+      place: mockedPlace,
+    })
+    render(<VenueMapModule />)
+
+    expect(screen.queryByText('Carte des lieux culturels')).not.toBeOnTheScreen()
+  })
+
+  it('should render venue map block when user is located everywhere', () => {
```
Comment (cedricls1) [inline src/features/home/components/modules/VenueMapModule.native.test.tsx:54]: `should not`

File: src/features/home/components/modules/VenueMapModule.web.test.tsx [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { VenueMapModule } from 'features/home/components/modules/VenueMapModule'
+import { render, screen } from 'tests/utils/web'
+
+describe('VenueMapModule', () => {
+  it('should not display venue map block when is "web"', () => {
```
Comment (cedricls1) [inline src/features/home/components/modules/VenueMapModule.web.test.tsx:7]: `when platform is "web"` ? J'ai l'impression qu'il manque un mot

File: src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx [hunk 5]
```diff
@@ -354,10 +354,13 @@ describe('<SearchListHeader />', () => {
...
         },
       })
-      mockUseLocation.mockReturnValueOnce({
+      const location = {
         geolocPosition: mockPosition,
         selectedLocationMode: LocationMode.AROUND_ME,
-      })
+        hasGeolocPosition: true,
+      }
+      mockUseLocation.mockReturnValueOnce(location)
```
Comment (cedricls1) [inline src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx:362]: Un seul ne suffit pas ?

File: src/features/venueMap/hook/useShouldDisplayVenueMap.native.test.ts [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+}))
+
+describe('useShouldDisplayVenueMap', () => {
+  it('should render venue map when user is located and feature flag enabled', () => {
+    const { result } = renderHook(useShouldDisplayVenueMap)
+
+    expect(result.current).toBe(true)
+  })
+
+  it('should render venue map when feature flag is disabled', () => {
```
Comment (cedricls1) [inline src/features/venueMap/hook/useShouldDisplayVenueMap.native.test.ts:34]: `should not`? (pour les 2 tests qui suivent aussi)

File: src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx (hunk 5 above)
Comment (voisinhugo) [reply src/features/search/components/SearchListHeader/SearchListHeader.native.test.tsx:362]: Nan malheureusement… Parce qu'on a un appel dans <SearchListHeader/> et dans le nouveau hook

**Review by cedricls1 — APPROVED** [review]

**Pull request merged**